weblate/checks/flags.py

```diff
@@ -283,7 +285,9 @@ def _format_values(self):
         return (self.format_flag(item) for item in self._items.values())
 
     def format(self) -> str:
-        return ", ".join(sorted(self._format_values()))
+        return format_html_join_comma(
+            "{}", ((mark_safe(flag),) for flag in sorted(self._format_values()))
```

Comment:
Ruff wants to remove mark_safe() but the tests fail if I remove it because the quotes get html escaped. Is it safe to use mark_safe here?

Reply:
This one should not be localized at all as it used to serialize flags into the database.